screenpy/speech_tools.py

```diff
@@ -32,7 +34,7 @@ class name: replace each capital letter with a space and a lower-case
         str: the string to place within another string.
     """
     if isinstance(describable, Describable):
-        description = describable.describe()  # type: ignore # see PEP 544
+        description = describable.describe()
```

Comment:
I don't know what happened, but this is no longer a type violation.

Reply:
mypy update must have fixed that.